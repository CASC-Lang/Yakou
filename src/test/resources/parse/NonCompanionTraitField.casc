trait NonCompanionTraitField {
    a: i32
}