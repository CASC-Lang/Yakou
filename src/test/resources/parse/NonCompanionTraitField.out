error: Cannot declare non-companion field a in trait
--> ./NonCompanionTraitField.casc:2:4
1 | trait NonCompanionTraitField {
2 |     a: i32
        ^= hint: Declare this field in companion block
3 | }